DomainContent/Mexico/createSpirit.js

```diff
@@ -113,29 +115,28 @@
 
     function getPositionToCreateEntity() {
         var direction = Quat.getFront(MyAvatar.orientation);
-        var distance = 0.3;
+        var distance = DISPLACEMENT;
```

Comment:
Do you still need this lowercase distance variable or can you just multiple the direction by the DISPLACEMENT variable?